### PR TITLE
Modal popup

### PR DESCRIPTION
**Pull request by kennyegun24**
Added modal popup 
Added work section dynamically
Added popup dynamically
Styled popup
Removed popup and works from `html`

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -22,3 +22,148 @@ spreadList.forEach((list) => {
...
+  description2: 'Just a dummy webpage initially created to be my portfolio, created with only HTML and javascript with no template guide. Zero functionality. Just a dummy webpage, on interractive.',
+  description: 'Just a dummy webpage initially created to be my portfolio, created with only HTML and javascript with no template guide. Zero functionality. Just a dummy webpage, on interractive.',
+  technology: ['Bootstrap', 'Html', 'CSS'],
+  sourceCode: '',
+  liveDemo: '',
+},
+];
+
+const modalSection = document.querySelector('.prof');
+const card = document.querySelector('.biew');
```
Comment (leonardodiasb) [inline main.js:96]: - [ ] Your modal looks great, but on some mobile screens, it does not contain the entire content leaving the buttons to be displayed out of the content like on the Iphone SE. Please make sure your modal is responsive and it displays correctly on every screen size.
 -Iphone SE view:
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @kennyegun24, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Good use of GitHub flow. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (kennyegun24) [reply main.js:96]: Thank you for pointing this out. I am already working on it

Comment (kennyegun24) [reply main.js:96]: Thanks, I have already worked on it and also fixed the view on 360px screen sizes like Samsung mobile screens

**Review by Reem-lab — APPROVED** [review]
# Hi @kennyegun24,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯



## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality✅
📌 Good commit messages ✅
📌 Professional README ✅

## Optional suggestions
- [ OPTIONAL ] I recommend fixing it in more screen size to let the whole pop up appear since in some phone screen as well I can't see it 😊🚀

- [ OPTIONAL ] I suggest to make a folder and put all your images inside it to get well structured files 🚀😊💯

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kennyegun24** [comment]
HI @Reem-lab, thanks for the correction, I have fixed it now, you can help me check to see if it is okay 😊😁

**Comment by Reem-lab** [comment]
Hi @kennyegun24 I checked it, and I think it is great now, Good job 🚀😊🔥🔥🔥

**Pull request merged**